.github/workflows/php-test.yml

```diff
@@ -39,9 +39,15 @@ on:
 
 jobs:
   php-test:
-    name: PHP
+    name: "PHP ${{ matrix.php }}"
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.4','8.0','8.1']
```

Comment:
Ideally we also test PHP 7.0, 8.2, and 8.3 (as experimental); not sure if wp-env supports all of these